pkg/operator/operator.go

```diff
@@ -44,10 +47,41 @@ const (
 	cronInterval          = 5  // seconds
 )
 
-var markedWorkflows = strset.New()
+var (
+	awsClient       *aws.Client
+	telemtryClient  *telemetry.Client
+	markedWorkflows = strset.New()
+)
 
 func main() {
-	telemetry.ReportEvent("operator.init")
+	conf := config.NewFromEnv()
```

Comment:
Maybe it should be called `config.Init()`, and not return anything, since in the rest of the codebase we use the `config` package directly?